### ISSUE TITLE
[Bug Report] Maximum call stack size exceeded

### ISSUE THREAD

**Issue by jrvaja** [vuetifyjs/vuetify#5633]
### Versions and Environment
**Vuetify:** 1.3.0
**Vue:** 2.5.17
**Browsers:** Chrome 69.0.3497.100
**OS:** Mac OS 10.13.6

### Steps to reproduce
- Click on UI Components -> Inputs & Controls -> forms
- Click on any other nav-drawer like (UI Components -> Inputs & Controls -> selects)

Now everything is stucked. We need to refresh the page.

https://www.dropbox.com/s/7gmowlc6lcb780i/Screenshot%202018-10-18%2016.15.09.png?dl=0

### Expected Behavior
Fix the solution

### Actual Behavior
I was working with 1.2.2

### Reproduction Link
You need to follow the steps, it could not display by any reproduction link.

### Other comments
Request for fix

**Comment by DrCoc** [comment]
Hi @QuanticPotatoes, 
I have also this issue. I tried to replace the `mergeData` function with the code of the merge request but the problem persist. 

It seems to come from v-tabs item. See the following screenshot:
![](https://files.slack.com/files-pri/T33L337G9-FE56U0Q4R/capture_d___e__cran_2018-11-16_a___18.25.02.png)

Hope for an update too 😄

**Comment by supertino7** [comment]
Hi, 
Just came upon this topic, googling "Maximum call stack size exceeded", which is happening on my Nuxt project.
After trying to isolate the problem, it seems to come from the `v-tabs` component, or more exactly from the `v-tab-items` and `v-tab-item` component. 

In my component, I used the sample code from the example on the official doc

```
<v-dialog fullscreen transition="dialog-bottom-transition" v-model="$store.state.showDialog">
  <v-toolbar color="cyan" dark tabs>
      <v-toolbar-side-icon></v-toolbar-side-icon>
      <v-toolbar-title>Page title</v-toolbar-title>
      <v-spacer></v-spacer>
      <v-btn icon>
        <v-icon>close</v-icon>
      </v-btn>
      <v-tabs
        slot="extension"
        v-model="activeSection"
        centered
        color="cyan"
        slider-color="yellow"
      >
        <v-tab
          v-for="i in 3"
          :key="i"
          :href="`#tab-${i}`"
        >
          Item {{ i }}
        </v-tab>
      </v-tabs>
    </v-toolbar>
    <v-tabs-items v-model="activeSection">
      <v-tab-item
        v-for="i in 3"
        :id="`tab-${i}`"
        :key="i"
      >
        <v-card flat>
          <v-card-text>Just met</v-card-text>
        </v-card>
      </v-tab-item>
    </v-tabs-items>
</v-dialog>
```

Got this error each time I refresh the windows, or with HMR when I edit store files **but not** when I edit  .vue file (page or component) 
```
RangeError: Maximum call stack size exceeded
    at mergeData (C:\Sources\project\node_modules\vue\dist\vue.runtime.common.js:1126:20)
```

If I remove `v-tabs-items` (or `v-tab-item`) from the template, it works.
Can this issue be related to #8967 ?

**Comment by KaelWD** [comment]
:man_facepalming: 
this issue is literally why I made that PR

**Comment by supertino7** [comment]
Thanks @KaelWD 
I replaced the `mergeData` function with the one in PR, but like @DrCoc it doesn't fix the issue :(

**Comment by supertino7** [comment]
> @supertino7
> Did you replace it in `Vue/dist/vue.runtime.common.js`?

No, only in `vue-server-renderer/build.js`. 
I tried in `vue.runtime.common.js` and now it seems to fix the issue
Thanks for your suggestion 👍

**Comment by jrvaja** [comment]
Hey @KaelWD 

Eagerly awaiting the fix for this issue.
Just gentle question, when do you plan for this?

Regards,

**Comment by jacekkarczmarczyk** [comment]
Should be already fixed in vue 2.5.18-beta.0

**Comment by jrvaja** [comment]
Hey @jacekkarczmarczyk ,

Could you please share vuetify 2.5.18-beta.0 document repo and production repo link?

Regards,
